README.md

```diff
@@ -70,9 +72,25 @@ console.log(errors) // => []
 
 XSD template includes E-form metadata and some basic types (EnumerationType, PrilohaType), see [template.xsd](forms/00603481.dopravneZnacenie.sk/template.xsd)
 
+### using CLI
+run `json-schema-xsd-tools <command> [options]`
```

Comment:
May depend on shell / system, but at least for me `npm install -g` does not add stuff to global path.

```suggestion
run `json-schema-xsd-tools <command> [options]` In case of `command not found`, try running using npx - `npx json-schema-xsd-tools <command> [options]`
```